README.md

```diff
@@ -22,6 +22,8 @@ Customization of Spark pods, e.g., mounting arbitrary volumes and setting pod af
 
 * Version >= 1.13 of Kubernetes to use the [`subresource` support for CustomResourceDefinitions](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#subresources), which became beta in 1.13 and is enabled by default in 1.13 and higher.
 
+* Version >= 1.16 of Kubernetes to use the MutatingWebhook/v1 and ValidatingWebhook/v1 of admissionregistration.k8s.io
```

Comment:
nit: to use `MutatingWebhook ` and `ValidatingWebhook ` `admissionregistration.k8s.io/v1`.

Reply:
Fixed the typo